HeRoBackEnd/Controllers/CandidateController.cs

```diff
@@ -46,45 +63,79 @@ public async Task<IActionResult> Get(int? id)
         //    return new JsonResult(new Candidate());
         //}
 
+        /// <summary>
+        /// Creates a candidate
+        /// </summary>
+        /// <param name="newCandidate">object of the NewCandidateViewModel class
+        /// containing information about the new candidate</param>
+        /// <returns>IActionResult</returns>
         [HttpPost]
+        [Route("Candidate/Create/{newCandidate.id}")]
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> Create(NewCandidateViewModel newCandidate)
         {
             //candidateService.Add(newCandidate);
 
             return RedirectToAction("Index");
         }
-
+        
+        /// <summary>
+        /// Updates information about a candidate
+        /// </summary>
+        /// <param name="candidateId">Id of the candidate</param>
+        /// <param name="newCandidate">Updated information about the candidate</param>
+        /// <returns>IActionResult</returns>
         [HttpPost]
+        [Route("Candidate/Edit/{candidateId}")]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Edit(int id, NewCandidateViewModel newCandidate)
+        public async Task<IActionResult> Edit(int candidateId, NewCandidateViewModel newCandidate)
         {
             //candidateService.Update(id, newCandidate);
 
             return RedirectToAction("Index");
         }
-
+        
+        /// <summary>
+        /// Deletes a candidate
+        /// </summary>
+        /// <param name="candidateId">Id of the candidate</param>
+        /// <returns>IActionResult</returns>
         [HttpPost]
+        [Route("Candidate/Delete/{candidateId}")]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Delete(int id)
+        public async Task<IActionResult> Delete(int candidateId)
         {
             //candidateService.Delete(id);
 
             return RedirectToAction("Index");
         }
-
+        
+        /// <summary>
+        /// Adds a candidate to the recruitment
+        /// </summary>
+        /// <param name="candidateId">Id of the candidate</param>
+        /// <param name="recruitmentId">Id of the recruitment</param>
+        /// <returns>IActionResult</returns>
         [HttpPost]
+        [Route("Candidate/AddRecruitment/{candidateId}")]
```

Comment:
Don't put anything in the query in this endpoint. Use post data to pass 2 ids

Reply:
Do you mean that I should't pass any parameters here in the route and just pass them as parametres to the method?

Reply:
Yes

Reply:
Ok, thanks